api/v1/server/embedded_spec.go

```diff
@@ -873,6 +873,10 @@ func init() {
       "description": "Response to a daemon configuration request. Contains the addressing\ninformation and configuration settings.\n",
       "type": "object",
       "properties": {
+        "Monitor": {
```

Comment:
Capitalization is not consistent. All properties start in lowercase.
Also, this is not consistent with the new "nodeMonitor" property in StatusResponse, which has the same type.
Use the "nodeMonitor" too here for consistency.